deploy/group_vars/citus_coordinator/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+pp_wal_level: hot_standby
+pp_max_wal_senders: 5
+pp_wal_keep_segments: 32
+pp_archive_mode: on
+pp_archive_command: 'cp %p {{ pp_datadir }}/archives/%f'
```

Comment:
@rioh It doesn't look like `pp_archive_mode` and `pp_archive_command` are used anywhere else.

Reply:
It's used in the postgres conf template.  Is that what you mean?

Reply:
Ah, sorry I had thought these were new settings, but you're just overriding them.  Makes sense!